ios/RCTCameraManager.m

```diff
@@ -539,6 +549,7 @@ - (void)captureStill:(NSInteger)target options:(NSDictionary *)options resolve:(
           reject(RCTErrorUnspecified, nil, RCTErrorWithMessage(error.description));
         }
       }];
+      }];
```

Comment:
Ah I see now that it's closing the `getDeviceOrientationWithBlock`, maybe it would be better to indent this whole block to match?